python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,90 @@
 .. _quantile_estimation_wilks:
 
-Estimation of a quantile by Wilks' method
------------------------------------------
+Estimation of a quantile upper bound by Wilks' method
+-----------------------------------------------------
 
-Let us denote
-:math:`\underline{Y} = h\left( \vect{X},\vect{d} \right) = \left( Y^1,\ldots,Y^{n_Y} \right)`,
-where :math:`\vect{X}= \left( X^1,\ldots,X^{n_X} \right)` is a random
-vector, and :math:`\vect{d}` a deterministic vector. We seek here to
-evaluate, using the probability distribution of the random vector
-:math:`\vect{X}`, the :math:`\alpha`-quantile :math:`q_{Y^i}(\alpha)` of
-:math:`Y^i`, where :math:`\alpha \in (0, 1)`:
+We consider a random variable :math:`X` of dimension 1 and its unknown quantile :math:`x_{\alpha}` of order :math:`\alpha`.
+We seek to evaluate an upper bound of :math:`x_{\alpha}` with a confidence greater than :math:`\beta`, using a given order statistics.
+
+We denote by :math:`(X_1, \dots, X_\sampleSize)` some independent and identically distributed variables according to :math:`X`.
+
+We denote by :math:`(X_1, \dots, X_\sampleSize)` some independent and identically distributed variables according to :math:`X`.
```

Comment:
This is duplicated.
```suggestion
```

Reply:
I do not agree because we need to introduce $(X_1, \dots, X_n)$ in order to define the order statistics just after:
 Let $X_{(k)}$ be the :k-th order statistics of $(X_1, \dots, X_n)$